tests/test_utils.py

```diff
@@ -14,6 +14,7 @@ def test_blacklist_url():
     :return: None
     """
 
+    original = epsagon.http_filters.BLACKLIST_URLS # Storing old state
```

Comment:
Extract to pytest.fixture like: 
```
original = epsagon.http_filters.BLACKLIST_URLS
yield
epsagon.http_filters.BLACKLIST_URLS = original
```
(instead of making it a part of this specific test)

Reply:
Done